packages/gestalt/src/Collection.js

```diff
@@ -45,6 +45,7 @@ import layoutStyles from './Layout.css';
 
 type Props = {|
   Item: ({ idx: number }) => React.Node,
+  itemPadding?: number,
```

Comment:
This can be calculated outside of `<Collection/>` - it's what the `paddingAll` functions in `<Collage/>` do.

Reply:
The width and height of the item layer in `<Collection>` are currently max of x-of-right-edge and max of y-of-bottom-edge. It includes the left / top border gutters but excludes the right / bottom border gutters.

For example, in the collage below, the Collection item layer would have width and height the same as what the red box shows:
<img width="323" alt="collection-size" src="https://user-images.githubusercontent.com/5341184/43221664-5dc9675e-9002-11e8-9cb0-a4a57413dd3e.png">

While this doesn't actually cause any problems, I figured that it's probably best if we make sure the Collection size matches the Collage's size.

<img width="323" alt="collection-size-fixed" src="https://user-images.githubusercontent.com/5341184/43221863-102ba722-9003-11e8-8f33-13311a55eb5d.png">

I understand this additional logic might not be necessary. Happy to change it if you don't think Collection item layer size needs to match the Collage's.

Reply:
I think here you could account for the extra padding by passing in a bigger `width` & `height` or `viewportWidth`/`viewportHeight` to collection. It's behavior of maxing the items is only a convenient default.